p2panda-net/src/sync/handler.rs

```diff
@@ -42,10 +42,7 @@ impl SyncConnection {
         let _span = debug_span!("connection", connection_id, %remote_addr);
 
         // Create a bidirectional stream on the connection.
-        let (mut send, mut recv) = connection
-            .accept_bi()
-            .await
-            .map_err(|e| SyncError::Protocol(e.to_string()))?;
+        let (mut send, mut recv) = connection.accept_bi().await?;
```

Comment:
This used `SyncError` before but I'm not sure if we this is an actual error coming from the sync protocol itself. Connectivity issues should maybe be considered a separate issue and deserve their own error type?